packages/gatsby-remark-images/src/index.js

```diff
@@ -4,6 +4,7 @@ const {
   imageWrapperClass,
 } = require(`./constants`)
 const visitWithParents = require(`unist-util-visit-parents`)
+const getDefinitions = require(`mdast-util-definitions`)
```

Comment:
I think we need to explicitely add this to `package.json` (right now it works because `mdast-util-to-hast` installs it)